R/adjust.R

```diff
@@ -82,6 +83,12 @@ adjust_infection_to_report <- function(infections, delay_defs,
                                        reporting_model, reporting_effect,
                                        type = "sample",
                                        truncate_future = TRUE) {
+  #nolint start: missing_argument_linter
+  deprecate_warn(
+    "1.5.0",
+    "adjust_infection_to_report()"
```

Comment:
can we add a detail that this will be completely removed in 2.0.0 as for the other deprecations (see examples in https://github.com/epiforecasts/EpiNow2/blob/dist-interface/R/deprecated.R).

Reply:
Thanks. I've done that here bad9ea4aa848670a479abc7733dbe823362892ea.